packages/styles/scss/components/tabs/_tabs.scss

```
@@ -244,6 +244,24 @@
       text-align: left;
     }
 
+    //-----------------------------
+    // Icon Item
+    //-----------------------------
+
+    .#{$prefix}--tabs__nav-item--icon {
+      display: flex;
+    }
+
+    .#{$prefix}--tabs__nav-item--icon--default {
+      padding-right: $spacing-04;
+      padding-left: $spacing-04;
+    }
+
+    .#{$prefix}--tabs__nav-item--icon--lg {
+      // had to use exact pixels because there is no way to get to 48px h/w without it.
+      padding: 13px 14px;
```

I think `px` values are generally converted to `rem` elsewhere in the codebase

```suggestion
      padding: rem(13px) rem(14px);
```